src/main/editDelete/postProcess.ts

```diff
@@ -1,70 +1,70 @@
-import { Transcription, Clip } from 'sharedTypes';
+// import { Transcription, Clip } from 'sharedTypes';
```

Comment:
Just wanted to clarify what is going on in this file. Is the code commented out for reason? Is it going to make a comeback?